### PR TITLE
Add HTML file to documentation

### PR DESCRIPTION
**Pull request by alexandra-simeonova**
**Description**

Changes proposed in this pull request:

- adding simple example to getting started guide

**Related issue(s)**
#1080

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -25,13 +25,89 @@ This diagram illustrates the structure of Luigi:
...
+                            imgWidth: "300",
+                            imgTopMargin: true
+                        }
+                    },{
+                        pathSegment: 'hello2',
+                        label: 'Hello Mario',
+                        viewUrl: 'https://fiddle.luigi-project.io/examples/microfrontends/multipurpose.html',
+                        isolateView: true,
+                        context: {
+                            title: 'Hello mario!',
```
Comment (zarkosimic) [inline docs/getting-started.md]:
```suggestion
                            title: 'Hello Mario!',
```

File: docs/getting-started.md [hunk 2]
```diff
@@ -25,13 +25,89 @@ This diagram illustrates the structure of Luigi:
...
+	<script>
+        Luigi.setConfig({
+            navigation: {
+                nodes: [{
+                    pathSegment: 'home',
+                    hideFromNav: true,
+                    children: [{
+                        pathSegment: 'hello',
+                        label: 'Hello Luigi',
+                        viewUrl: 'https://fiddle.luigi-project.io/examples/microfrontends/multipurpose.html',
```
Comment (zarkosimic) [inline docs/getting-started.md]:
```suggestion
                        viewUrl: 'https://fiddle.luigi-project.io/examples/microfrontends/multipurpose.html',
                        isolateView: true,
```

Comment (zarkosimic) [reply docs/getting-started.md]: Due to error in the console.

File: docs/getting-started.md [hunk 3]
```diff
@@ -25,13 +25,90 @@ This diagram illustrates the structure of Luigi:
 
 <img src="https://github.com/SAP/luigi/blob/master/docs/assets/luigi-overview-diagram.jpg?raw=true" width="600"/>
 
+ Do you want to try out Luigi without installing it on your machine? Try one of these options:
+
+<!-- accordion:start -->
+
+ ### HTML file
+
+It is possible to create a Luigi application using a single HTML file. This example aims to show the simplicity of Luigi, but is not intended for real life use.
```
Comment (kazydek) [inline docs/getting-started.md]:
```suggestion
It is possible to create a Luigi application using a single HTML file. This example aims to show the simplicity of Luigi, but it is not intended for real-life use.
```

File: docs/getting-started.md [hunk 4]
```diff
@@ -25,13 +25,90 @@ This diagram illustrates the structure of Luigi:
...
+            }
+        });
+    </script>
+</body>
+
+</html>
+```
+
+### Luigi Fiddle
+Go to the [Luigi Fiddle](https://fiddle.luigi-project.io/) website. Click on `Modify Config` at the bottom right to make changes to the Luigi application.
```
Comment (kazydek) [inline docs/getting-started.md]:
```suggestion
Go to the [Luigi Fiddle](https://fiddle.luigi-project.io/) website. Click on **Modify Config** at the bottom right of the page to make changes to the Luigi application.
```

File: docs/getting-started.md [hunk 5]
```diff
@@ -25,13 +25,90 @@ This diagram illustrates the structure of Luigi:
...
+
+Copy and paste this code in a text editor, then save it as an HTML file:
+
+```html
+ <!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<title>Hello Luigi</title>
+	<link rel='stylesheet' href='https://unpkg.com/@kyma-project/luigi-core/luigi.css'>
```
Comment (maxmarkus) [inline docs/getting-started.md]: If I am super picky, i would say the intendation of titel and link rel is wrong :-)

**Pull request merged**